src/core/lua/luacfb.h

```diff
@@ -33,7 +33,7 @@ class CLuaCFB : public CGenFunctionBlock<CCompositeFB> {
     bool initialize() override;
 
     bool createInterfaceSpec(const char *, SFBInterfaceSpec &paInterfaceSpec) override {
-      paInterfaceSpec = *mInterfaceSpec;
+      paInterfaceSpec = mInterfaceSpec;
```

Comment:
this should be:
`paInterfaceSpec = *paTypeEntry->getInterfaceSpec()`

Reply:
same as above

Reply:
Ok, done